src/problem_bank_scripts/tui/tui.py

```diff
@@ -88,6 +87,20 @@ def generate_given_choices(options: list[str], answer: str, question: str, use_g
     "symbolic-input": {},
 }
 
+def convert_solution_to_type(solution: str, key: str):
+    if key == "number-input":
+        # remove ',' from solution
+        return remove_edge_non_numeric(solution.replace(",", ""))
+    elif key == "integer-input":
+        return remove_edge_non_numeric(solution.replace(",", ""))
```

Comment:
```suggestion
    if key in {"number-input", "integer-input"}:
        # remove ',' from solution
        return remove_edge_non_numeric(solution.replace(",", ""))
```
I don't see a difference on these codepaths?